src/coins.cpp

```diff
@@ -1283,6 +1302,17 @@ bool CCoinsViewCache::IsScTxApplicableToState(const CTransaction& tx, libzendoom
         if (!CheckScTxTiming(scId))
             return false;
 
+        /**
+         * Check that the Forward Transfer amount is greater than or equal to the
+         * Sidechain Forward Transfer Fee.
+         */
+        CAmount scFtFee = GetActiveCertView(scId).forwardTransferScFee;
+        if (ft.nValue < scFtFee)
```

Comment:
From MC perspective it seems fine that we allow `ft.nValue == scFtFee`, but from SC perspective seems strange.
If amounts are equal, all the coins will be passed as fees and no Box will be created (or a Box with 0 value which is a mess). This case will demand some kind of a workaround on SDK side.
Let's discuss it.

Reply:
Changed to code to check that FT value is strictly greater than sidechain FT fee.